### ISSUE TITLE
View other map levels after the correct answer is guessed

### ISSUE THREAD

**Issue by kenneyd** [brendaninnis/satle#15]
I started playing recently, and I finished the recent Osaka in 1, and Moscow in 2, but for Perth I think I basically used all my tries. For Perth though, it was cool seeing the various levels of the city, and zooming back in through each one afterwards, but you can't do this if you guess the right answer too quickly. I think it would be nice to open up all the zoom levels after guessing the correct answer.

**Comment by brendaninnis** [comment]
Great idea! I think I could just enable all the map controls once the game is over. Then players can pan and zoom the map however they want.

**Comment by CatzHoekk** [comment]
Yeah, that'd be nice. You really have an urge to zoom in again and check out the area.
